dashboard/app/models/stage.rb

```diff
@@ -127,6 +127,37 @@ def summarize
     stage_summary.freeze
   end
 
+  # Provides a JSON summary of a particular stage, that is consumed by tools used to
+  # build lesson plans
+  def summary_for_lesson_plans
```

Comment:
It still seems not great to have this method in addition to `Stage#summarize` but I'm not sure how I'd de-duplicate.